EXPLAINER.md

```diff
@@ -642,3 +642,10 @@ Some other notes:
 See
 [security_and_privacy.md](https://github.com/webscreens/window-placement/blob/master/security_and_privacy.md)
 for additional explorations of privacy and security concerns.
+
+## Related explainers:
+| Name | Link |
+|------|------|
+| Window Segments Enumeration API | [Explainer](https://github.com/webscreens/window-segments/blob/master/EXPLAINER.md)|
```

Comment:
Should we also cite the 'spanning' CSS media feature and device fold CSS environment variables ([Explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Foldables/explainer.md))?

Reply:
(ditto in the readme)

Reply:
Agree, I think we should, I will update that in both our explainers later this afternoon.